src/CSET/recipes/lfric_generic_domain_mean_vertical_profile_series.yaml

```diff
@@ -0,0 +1,41 @@
+category: Quick Look
+title: Domain mean horizontal $VARNAME vertical profile as series
+description: Plots a time series of the domain mean surface $VARNAME.
```

Comment:
Same comment as previous recipe

Reply:
Changed to:
title: Domain mean $VARNAME vertical profile as series
description: Plots a time series of the vertical profile of domain mean $VARNAME using a log pressure coordinate.